auth-service/src/main/resources/templates/thymeleaf/fragments/template.html

```diff
@@ -17,7 +17,7 @@
     <link href="/static/assets/images/govuk-apple-touch-icon-152x152.png" rel="apple-touch-icon"
           sizes="152x152">
     <link href="/static/assets/images/govuk-apple-touch-icon.png" rel="apple-touch-icon">
-    <link href="/static/govuk-frontend-4.8.0.min.css" rel="stylesheet">
```

Comment:
And should govuk-frontend-4.8.0 be deleted from the repo?

Reply:
@hugh-emerson 
Ideally yes. I don't think we need that old file anymore. I will do that now.